src/screens/ConfirmationCode/ConfirmationCodeScreen.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState, useEffect } from 'react';
+import { View, Text } from 'react-native';
+import { Auth } from 'aws-amplify';
+import { CommonActions, useNavigation } from '@react-navigation/native';
+import Input from '../../components/Input/Input';
+import CommonButton from '../../components/CommonButton/CommonButton';
+import * as SCREEN from '../../constants/screen.constant';
+import { toastSuccess, toastError } from '../../utils/toast.utils';
+import styles from './styles';
+
+const ConfirmationCodeScreen = ({ route }) => {
+  const navigation = useNavigation();
+  const email = route.params.email;
+  const [confirmationCode, setConfirmationCode] = useState('');
+  const [loading, setLoading] = useState(false);
+  const [error_message, setErrorMessage] = useState('');
+
+  const confirmationCodeHandler = text => {
+    setConfirmationCode(text);
+    console.log(confirmationCode);
+  };
+
+  const handleConfirm = async () => {
+    try {
+      setLoading(true);
+      setErrorMessage('');
+      const authResponse = await Auth.confirmSignUp(email, confirmationCode);
```

Comment:
Steps to follow:

1. After having the `confirmSignUp`, call the `login` auth of amplify.
2. Save the access token returned from the `login`.
3. Redirect the user to home (which is in `SCREEN.APP_NAVIGATOR`)